src/js/components/DescriptionList.js

```diff
@@ -1,6 +1,12 @@
+import PureRenderMixin from 'react-addons-pure-render-mixin';
 import React from 'react';
 
 class DescriptionList extends React.Component {
+  constructor() {
+    super(...arguments);
+    this.shouldComponentUpdate = PureRenderMixin.shouldComponentUpdate;
```

Comment:
I was just looking here: https://facebook.github.io/react/docs/pure-render-mixin.html
Isn't it supposed to have access to `this.props` and `this.state`?